rate-limiter/src/main/scala/it/pagopa/interop/commons/ratelimiter/model/Headers.scala

```diff
@@ -21,7 +23,8 @@ object Headers {
     List(
       header(contexts, RATE_LIMITER_HEADER_LIMIT),
       header(contexts, RATE_LIMITER_HEADER_INTERVAL),
-      header(contexts, RATE_LIMITER_HEADER_REMAINING)
+      header(contexts, RATE_LIMITER_HEADER_REMAINING),
+      getClaim(contexts, ACCEPT_LANGUAGE).toOption.map(RawHeader(CONTENT_LANGUAGE, _))
```

Comment:
the function in this module is intended only for rate limiting.
To return the header unfortunately I think that we still need to change the single service.
We can limit this change on the BFF for now